index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
+    </div>
+  </header>
+  <section class="section-text">
+    <div class="section-title"><h1>I'm Daniel <br>Glad to see <br>you!</h1></div>
+    <div class="section-content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+  </section>
+  <footer>
+    <div class="footer-content">LET'S CONNECT</div>
+    <div class="main-footer-network-content">
+      <div class="footer-network twitter"><img src="Images/twitter.svg" alt="twitter"></div>
+      <div class="footer-network linkedin"><img src="Images/linkedin.svg" alt="linkedin"> </div>
+      <div class="footer-network M"> <img src="Images/m.svg" alt="M"> </div>
+      <div class="footer-network github"> <img src="Images/github.svg" alt="github"> </div>
+      <div class="footer-network hand"> <img src="Images/hand.svg" alt="hand"> </div>
```

Comment:
- [ ] please ensure that your social icons are inside a `ul tag` and each of them in a `li tag`.
- [ ] Add link place holders to each icon ..`<a href="/"></a>` .you can check how to add that [here](https://www.w3schools.com/html/html_links.asp)